src/session/keys.ts

```diff
@@ -20,3 +20,7 @@ export const LOCALE: string = "locale";
 export const PERMISSIONS: string = "permissions";
 export const SCOPE: string = "scope";
 export const COMPANY_NUMBER: string = "company_number";
+export const PTF_SESSION: string = "ptf_session";
+export const PTF_REQUESTS: string = "ptf_requests";
```

Comment:
I don't think we need PTF_REQUESTS (yet).

Reply:
Removed.